src/challenges.js

```diff
@@ -2,17 +2,33 @@
 const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);
+  }
 };
 
 const map = (elements, cb) => {
   // Produces a new array of values by mapping each value in list through a transformation function (iteratee).
   // Return the new array.
```

Comment:
This is a great solution.  If you come back to this try and solve this using the `each` function you created above (just a little extra credit/thinking)